packages/react/toast/src/Toast.tsx

```diff
@@ -542,11 +547,6 @@ const ToastImpl = React.forwardRef<ToastImplElement, ToastImplProps>(
                 })}
               >
                 <Primitive.li
-                  role="status"
-                  aria-live={type === 'foreground' ? 'assertive' : 'polite'}
-                  aria-atomic
```

Comment:
this stuff should exist still but with `aria-live="off"` so that SRs read aloud that it is a status list or status when focused on a particular toast.  without this, it communicates it as a group list or group when focused on a particular toast.  

You can test the following in VO to see what i mean:

- current version: https://codesandbox.io/s/dawn-cherry-6iwxpm?file=/src/App.js
- previous version: https://codesandbox.io/s/sad-stallman-53xhxb?file=/src/App.js

Reply:
Ah thanks, so basically this is to announce as "status" rather than "group"?

I guess that answers this question then! https://github.com/radix-ui/primitives/pull/1528#discussion_r920923915

Reply:
> Ah thanks, so basically this is to announce as "status" rather than "group"?

yep 👍